package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.3.5",
   "description": "AST utility module for statically analyzing JSX",
   "main": "lib/index.js",
+  "files": [
+    "lib"
```

Comment:
The hazard with "npmignore" is that you'd accidentally publish unneeded files. The hazard with "files" is that you'd accidentally fail to publish a needed file.

An exclusion list is the safer approach here.